### PR TITLE
Disable nginx's access log for obelisk server

### PR DESCRIPTION
**Pull request by 3noch**
Otherwise nginx will save access logs to disk which can fill up the disk

I have:

  - [x] Based work on latest `develop` branch
  - [ ] Run the test suite: `$(nix-build -A selftest --no-out-link)`
  - [ ] (Optional) Run CI tests locally: `nix-build release.nix -A build.x86_64-linux --no-out-link` (or `x86_64-darwin` on macOS)

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -138,6 +138,9 @@ in rec {
             proxyPass = "http://127.0.0.1:" + toString internalPort;
             proxyWebsockets = true;
           };
+          extraConfig = ''
+            access_log stdout;
+          '';
```
Comment (3noch) [inline default.nix:143]: Maybe should put this inside location block instead.

File: default.nix [hunk 2]
```diff
@@ -137,6 +137,9 @@ in rec {
           locations.${baseUrl} = {
             proxyPass = "http://127.0.0.1:" + toString internalPort;
             proxyWebsockets = true;
+            extraConfig = ''
+              access_log stdout;
```
Comment (matthewbauer) [inline default.nix:141]: not sure how much this will spam your syslog, but i think it will just be info severity so that's okay

Comment (matthewbauer) [reply default.nix:141]: let's just set it to off, since obelisk has its own access log

Comment (3noch) [reply default.nix:141]:
```suggestion
              access_log off;
```

**Pull request merged**